packages/react-components/react-list-preview/stories/List/MultipleActionsSelection.stories.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  Button,
+  Caption1,
+  makeResetStyles,
+  makeStyles,
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  mergeClasses,
+  shorthands,
+  Text,
+  tokens,
+} from '@fluentui/react-components';
+import { List, ListItem, ListProps } from '@fluentui/react-list-preview';
+import { MoreHorizontal20Regular } from '@fluentui/react-icons';
+
+import * as React from 'react';
+
+const useListItemRootStyles = makeResetStyles({
+  position: 'relative',
+  flexGrow: '1',
+  gap: '8px',
+  border: '1px solid grey',
+  alignItems: 'center',
+  borderRadius: '8px',
+  gridTemplate: `"preview preview preview" auto
+      "header action secondary_action" auto / 1fr auto auto
+    `,
+});
+
+const useStyles = makeStyles({
+  list: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...shorthands.gap('16px'),
+    maxWidth: '300px',
+  },
+  listItem: {
+    display: 'grid',
+    ...shorthands.padding('8px'),
+  },
+  caption: {
+    color: tokens.colorNeutralForeground3,
+  },
+  image: {
+    maxWidth: '100%',
+    ...shorthands.borderRadius('5px'),
+  },
+  title: {
+    fontWeight: 600,
+    display: 'block',
+  },
+  checkmark: {
+    position: 'absolute',
+    left: '10px',
+    top: '10px',
+    zIndex: 1,
+  },
+  preview: {
+    ...shorthands.gridArea('preview'),
+    ...shorthands.overflow('hidden'),
+  },
+  header: { ...shorthands.gridArea('header') },
+  action: { ...shorthands.gridArea('action') },
+  secondaryAction: { ...shorthands.gridArea('secondary_action') },
+});
+
+const CustomListItem = (props: { title: string; value: string; selected?: boolean }) => {
```

Comment:
```suggestion
const CustomListItem = (props: { title: string; value: string }) => {
```

`selected` prop is not used